master/internal/restore.go

```diff
@@ -83,6 +83,11 @@ func (m *Master) restoreExperiment(expModel *model.Experiment) error {
 	taskContainerDefaults := m.getTaskContainerDefaults(poolName)
 	taskSpec := *m.taskSpec
 	taskSpec.TaskContainerDefaults = taskContainerDefaults
+	owner, err := m.db.UserByUsername(expModel.Username)
+	if err != nil {
+		return errors.Wrap(err, "invalid user")
```

Comment:
error may not be "invalid user"
```suggestion
		return errors.Wrapf(err, "retrieving full user on restart")
```